### DISCUSSION TITLE
DAUD audio recording off function. not work)

### DISCUSSION THREAD

**Discussion by SergKozlik** [gopro/labs#1152] [Q&A] [answered]
hi guys!

gopro 12, want to turn off sound recording for video

installed Goprolab, found in the description that DAUD 1 should turn off the sound. 
made a qr, and scanned with the camera. the camera says - the DAUD function is active.

Sound is recorded)). I've already racked my brains. What am I doing wrong?

Thanks!

**Comment by dnewman-gpsw** [comment]
It is currently not supported on HERO12. In the documentation the compatibility is listed as **DAUD H10-11/13**, and https://gopro.github.io/labs/control/extensions/ it says **not supported on HERO12**.  The good news is this was only a bug, and the update due in a few days will fix DAUD support.

**Reply by SergKozlik** [reply]
oh! My dear man!

This is a funny situation, when due to my carelessness I broke my head, literally. It is difficult to imagine such an exception for this model))) .

I really hope that this will be fixed.

I have already contacted support with a request to add an audio recording disable, sometimes this is a problem that requires additional time from the user to remove the sound by post-processing. And this reduces the applicability of the camera, with large volumes of filmed content. For example, for stockers, the question is quite serious. Hundreds of videos a day is an unacceptably large amount of unnecessary work.

but I was not heard by the company, unfortunately.

Thank you very much!

ps. How do I know when the GoProLab update is released?

**Reply by dnewman-gpsw** [reply, marked as answer]
Frustratingly you have have to manually check, there is no notifications for Labs updates. Visit https://gopro.github.io/labs/ in a week and it should be there.

**Comment by SergKozlik** [comment]
thanks a lot!
